core/domain/user_jobs_continuous.py

```diff
@@ -549,21 +550,45 @@ def map(item):
         else:
             calculate_exploration_impact_score = False
 
-        statistics = (
-            stats_jobs_continuous.StatisticsAggregator.get_statistics(
-                item.id, stats_jobs_continuous.VERSION_ALL))
-        answer_count = 0
-        # Find number of users per state (card), and subtract no answer
-        # This will not count people who have been back to a state twice
-        # but did not give an answer the second time, but is probably the
-        # closest we can get with current statistics to "number of users
-        # who gave an answer" since it is "number of users who always gave
-        # an answer".
-        for state_name in statistics['state_hit_counts']:
-            state_stats = statistics['state_hit_counts'][state_name]
-            first_entry_count = state_stats.get('first_entry_count', 0)
-            no_answer_count = state_stats.get('no_answer_count', 0)
-            answer_count += first_entry_count - no_answer_count
+        if feconf.ENABLE_NEW_STATS_FRAMEWORK:
+            exploration_stats_dict = stats_services.get_exploration_stats(
+                item.id, item.version)
+            # The domain object is passed in always. In the tests, we pass the
+            # dict directly instead.
+            if not isinstance(exploration_stats_dict, dict):
```

Comment:
This is very hacky. Don't do it. You'll need to find a way to create a domain object in the tests.